salt

```diff
@@ -96,7 +96,9 @@ fi
 
 # collect some sys info
 F_LOG "OS: $(cat /etc/*-release)"
-F_LOG "OS (lsb): $(lsb_release -a)"
+if [ $(which lsb_release 2> /dev/null) ]; then
```

Comment:
Please change that to something like:

`which lsb_release > /dev/null 2>&1`

as otherwise you will always get the lsb_release output twice